IAST Security Controls - dotnet (#28805)
DataDog/documentation: content/en/security/code_security/iast/security_controls/_index.md

`SANITIZER:SQL_INJECTION:node_modules/sql-sanitizer/index.js:sanitize`

{{% /collapse-content %}}

{{% collapse-content title=".NET" level="h4" %}}

> ## General syntax

Reviewer: ```suggestion
### General syntax
```

> `KIND:VULNERABILITIES:Assembly:Class:Method(ParameterTypes)[:ParameterIndexes]`

Reviewer 2: I would follow the same formatting and wording as the [format definition above](https://github.com/DataDog/documentation/blob/5a361c8667a4932d9822bfc52e8de0bca6b04308/content/en/security/code_security/iast/security_controls/_index.md?plain=1#L26).

KIND => TYPE
VULNERABILITIES => SECURE_MARKS

Author: Fine enough. Thankyou

<div class="alert alert-info">

> Parameter types must be fully qualified with their namespace. Example: `System.String`

Reviewer: ```suggestion
Parameter types must be fully qualified with their namespace. Example: <code>System.String</code><br /><br />
```
Some arcane formatting, brought to you by trial and error, to get the div to play a little more nicely 🙂

> Parameter indexes are comma separated. If no parameter index is provided, value is defaulted to 0 (first parameter)

Reviewer: ```suggestion
Parameter indexes are comma-separated. If no parameter index is provided, the value of the first parameter defaults to 0.
```
